src/components/molecules/userInfo/UserInfo.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './UserInfo.css'
+
+export default function UserInfo(props) {
+  return (

```

Comment:
요런식으로 한번 구조분해 해서 가져가도 좋을 것 같아요!

```suggestion
export default function UserInfo(props) {
  const { userName, repoCount, starCount } = props;

  return (

```